preternatural-build/action.yml

```diff
@@ -4,17 +4,17 @@ inputs:
   derived_data_path:
     description: 'The path to the derived data folder'
     required: false
-  build_all_platforms:
-    description: 'Builds for all supported platforms'
+  platforms:
+    description: 'Target platforms (comma-separated: ios,macos,tvos,watchos,visionos,all)'
     required: false
-    default: 'false'
+    default: 'macos'
   xcode-version:
     description: 'Xcode version to use'
     required: true
-  configuration:
-    description: 'Build configuration (debug or release)'
+  configurations:
+    description: 'Build configurations (comma-separated: debug, release, debugAndRelease)'
     required: false
-    default: 'debug'
+    default: 'debugAndRelease'
```

Comment:
I deleted debugAndRelease. It is not an array but a comma-separated string because we can pass that as is like `preternatural build --configurations debug,release`.